pkg/controller/keycloakbackup/keycloakbackup_controller.go

```diff
@@ -161,7 +161,7 @@ func (r *ReconcileKeycloakBackup) ManageError(instance *kc.KeycloakBackup, issue
 	}
 
 	return reconcile.Result{
-		RequeueAfter: RequeueDelayErrorSeconds * time.Second,
+		RequeueAfter: RequeueDelayError * time.Second,
```

Comment:
@slaskawi: In this row the "* time.Second" can be removed, because it was defined in the const section already.

Reply:
That's correct! You found a serious bug in the code! Thanks for spotting this.

Reply:
Updated.